pages/en/lb4/Sequence.md

```diff
@@ -13,30 +13,6 @@ summary:
 A `Sequence` is a stateless grouping of [Actions](#actions) that control how a
 `Server` responds to requests.
 
-```js
-const server = await app.getServer(RestServer);
```

Comment:
was this duplicate code?

Reply:
The code didn't work when I tried to copy paste it; the spec given in `sequence.handler` has the wrong error supposedly, and it seems like the code was refactored so that this wouldn't work anymore.